cross/stlink/Portfile

```diff
@@ -1,25 +1,28 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
-PortGroup           cmake 1.1
-PortGroup           github 1.0
+PortSystem             1.0
```

Comment:
the whitespace changes you make here do not follow the modeline: it should be multiples of four (so you cannot start at character 24). Additionally, if you want to make whitespace changes these should be in a separate commit from functional changes.

Reply:
Changed, thanks for your review!